tests/test.cfg

```diff
@@ -0,0 +1,304 @@
+#  Example Qudi configuration file.
```

Comment:
Maybe this should say it's a _test_ config file rather than an _example_ config file.